vernac/obligations.ml

```diff
@@ -20,6 +20,12 @@ open Pp
 open CErrors
 open Util
 
+(* EJGA: This should go away, no more need for fix_exn  *)
+module Stm = struct
+    let u = (fun x -> x)
+    let get_fix_exn () = u
+end
+
```

Comment:
I would (naively) think that the Stm would "fix" the exception upstream in this case, but as I wrote I need to give this issue careful thinking; plenty of time before the merge window closes.